pyproject.toml

```diff
@@ -48,7 +48,8 @@ test = [
     "jupyter_server_fileid[test]",
     "pytest>=7.0",
     "pytest-cov",
-    "websockets"
+    "websockets",
+    "importlib_metadata >=3.6; python_version<'3.10'",
```

Comment:
Why are you adding the `python_version` pin here?

Reply:
Python 3.10+ has [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html), otherwise we need [importlib_metadata](https://github.com/python/importlib_metadata).

Reply:
Ah, I see. I misunderstood what this line was doing. 

This is bringing in an optional dependency _if_ the current python version is < 3.10. 

I thought it was saying the `test` install can _only_ happen in python runtimes < 3.10.